<launch>
  <node name="webcam" pkg="usb_cam" type="usb_cam_node" >
    <param name="pixel_format" value="yuyv" />
  </node>

  <node name="maVisu" pkg="image_view" type="image_view" output="screen" >
    <remap from="image" to="/webcam/image_raw" />
  </node>

  <node name="maVisuSift" pkg="image_view" type="image_view" output="screen" />

  <node name="image_converter" pkg="tp_ros_ensta_u2is" type="image_converter.py">
    <remap from="image_topic_in" to="/webcam/image_raw" />
    <remap from="image_topic_out" to="/maVisuSift/image" />
  </node>

</launch>
